plugins/woocommerce/tests/e2e-pw/tests/merchant/order-emails.spec.js

```diff
@@ -11,7 +12,7 @@ test.describe( 'Merchant > Order Action emails received', () => {
 	const adminEmail =
 		process.env.USE_WP_ENV === '1'
 			? 'wordpress@example.com'
-			: 'admin@woocommercecoree2etestsuite.com';
+			: ADMIN_USER_EMAIL ?? 'admin@woocommercecoree2etestsuite.com';
```

Comment:
I had to change this to make sure that this test would use the admin user email in the smoke test site.